conflict-template/HEAD

```diff
@@ -0,0 +1 @@
+ref: refs/heads/master
```

Comment:
Where did all these `conflict-template/*` files come from? I think they were added by mistake....

Reply:
@beardofedu oh shoot, you're right, another reason why `git add . ` is baaad 😞